### PR TITLE
Usage Notes

### PR DESCRIPTION
**Pull request by una**
Did a very brief audit of some of these styles to add usage notes regarding accessibility and usage.

Would love some feedback on these!

### CODE REVIEW

File: comparisons/components/image_slider/a11y.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Usage: Screen reader accessible, no user control over images.
```
Comment (robdodson) [inline comparisons/components/image_slider/a11y.txt]: I might not use the phrase screen reader accessible. You might say this is presentational and skipped by the screen reader. Using an empty string in your alt tag, `alt=""`, explicitly tells the screen reader to ignore the image. Kind of like saying "Hey this is just decoration, it's not super important for the user experience".

File: comparisons/forms/color_picker/a11y.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+Usage: Safe for production.
```
Comment (robdodson) [inline comparisons/forms/color_picker/a11y.txt:1]: Probably want to add a label to the input and submit button before saying it's safe for production (I'm just looking at the live site, not sure if you've already done this).

File: comparisons/forms/file_upload/a11y.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+Usage: Safe for production.
```
Comment (robdodson) [inline comparisons/forms/file_upload/a11y.txt:1]: Probably want to add a label to the two controls, like "Upload a profile photo" or something like that

File: comparisons/navigation/accordion/a11y.txt [hunk 4]
```diff
@@ -0,0 +1 @@
+Usage: Safe for production.
```
Comment (robdodson) [inline comparisons/navigation/accordion/a11y.txt:1]: I don't think this one is safe for production. There's a lot of keyboard support and ARIA that goes into an accordion: https://www.w3.org/TR/wai-aria-practices-1.1/#accordion

File: comparisons/navigation/lightbox/a11y.txt [hunk 5]
```diff
@@ -0,0 +1 @@
+Usage: Not keyboard navigable.
```
Comment (robdodson) [inline comparisons/navigation/lightbox/a11y.txt:1]: I'd lump the lightbox in with the modal. Probably not safe to use as there's a lot of keyboard support that needs to be added

File: comparisons/navigation/tabs/a11y.txt [hunk 6]
```diff
@@ -0,0 +1 @@
+Usage:
```
Comment (robdodson) [inline comparisons/navigation/tabs/a11y.txt]: Similar to accordion, lots of keyboard and ARIA support needed https://www.w3.org/TR/wai-aria-practices-1.1/#tabpanel

**Comment by robdodson** [comment]
Left some comments on the a11y.txt bits, hope that helps!

**Comment by una** [comment]
Ahh you're the best @robdodson 😃✨ thanks!

**Pull request merged**